x-pack/plugins/cases/public/containers/api.ts

```diff
@@ -157,6 +162,22 @@ export const getReporters = async (signal: AbortSignal, owner: string[]): Promis
   return response ?? [];
 };
 
+export const getCaseMetrics = async (
+  caseId: string,
+  features: CaseMetricsFeature[],
+  signal: AbortSignal
+): Promise<CaseMetrics> => {
+  const response = await KibanaServices.get().http.fetch<CaseMetricsResponse>(
+    getCaseDetailsMetricsUrl(caseId),
+    {
+      method: 'GET',
+      signal,
+      query: { features: JSON.stringify(features) },
+    }
+  );
+  return decodeCaseMetricsResponse(response) as CaseMetrics;
```

Comment:
Do we need the convertToCamelCase call just in case? I don't believe we return anything in snake case at the moment.